pkg/common/hashmap/inthashmap.go

```diff
@@ -145,43 +145,48 @@ func fillKeys[T types.FixedSizeT](m *IntHashMap, vec *vector.Vector, size uint32
 }
 
 func fillStrKey(m *IntHashMap, vec *vector.Vector, start int, n int) {
-	bvs := vector.GetBytesVectorValues(vec)
+	area := vec.GetArea()
+	vs := vector.MustTCols[types.Varlena](vec)
 	keys := m.keys
 	keyOffs := m.keyOffs
 	if !vec.GetNulls().Any() {
 		if m.hasNull {
 			for i := 0; i < n; i++ {
+				v := vs[i+start].GetByteSlice(area)
```

Comment:
vec.GetByteSlice?  Is that slower?

Reply:
Inside GetByteSlice we always do a MustTCols, I am not sure if that is a performance problem.

Reply:
MustStrCols will call the following code, and this will result in additional memory allocation.
```
https://github.com/matrixorigin/matrixone/blob/784dc423027a263cf033e6264e62be63cf2d7c00/pkg/container/vector/tools.go#L46
https://github.com/matrixorigin/matrixone/blob/784dc423027a263cf033e6264e62be63cf2d7c00/pkg/container/types/bytes.go#L100
```